cookbook/security/voters.rst

```diff
@@ -92,6 +92,13 @@ the security layer. This can be done easily through the service container.
     methods in your implementation of the ``vote()`` method and return ``ACCESS_ABSTAIN``
     if your voter does not support the class or attribute.
 
+
```

Comment:
I think in this particular use case `VoterInterface` is actually the better choice.
`AbstractVoter` assumes you know exactly what attributes and classes you want to match, whereas in this situation we're meant to allow everything through (look at how `supportsAttribute` and `supportsClass` are defined).